scripts/clean.js

```diff
@@ -31,9 +31,12 @@ function cleanTsGeneratedFiles() {
         const tsconfig = JSON5.parse(readFileSync(filePath));
         const { outDir } = tsconfig.compilerOptions;
         if (outDir) {
-          const outPath = resolve(packagePath, outDir);
-          console.log(`Removing ${outPath}`);
-          removeSync(outPath);
+          // Skip over proto/lib, which contains source files
```

Comment:
Oh I see you addressed this here.
But this now means that the TS output cannot be cleaned with this script.

Reply:
Correct. I could allowlist the source files, or move them from `lib/` to `js/`?

Reply:
Let's use separate directories